amundsen_application/static/css/_variables-default.scss

```diff
@@ -1,48 +1,33 @@
-/**
- The following are used in Amundsen but are not part of Bootstrap.
- These must be defined here:
-
-  $brand-color-1, $brand-color-2, $brand-color-3, $brand-color-4, $brand-color-5
-
-  $font-family-header, $font-weight-header-regular, $font-weight-header-bold
-  $font-family-body, $font-weight-body-regular, $font-weight-body-semi-bold, $font-weight-body-bold
-
-  $text-dark, $text-medium, $text-light
-
-  $btn-primary-bg-hover, $btn-default-border-hover,
-  $btn-primary-border-hover, $btn-default-bg-hover,
-
-  */
-
+@import 'colors';
 
+// TODO - consider replacing brand-color-X
```

Comment:
What would it be replaced with?

Reply:
That's part of the consideration :). 

Currently we use `$brand-color-X` for various links, buttons, and icons, borders, backgrounds, etc. Sometimes these are referenced directly
```
button.edit-button:hover {
  background-color: $brand-color-4;
}
```

and sometimes referenced indirectly
```
$indigo60: #665AFF;
$brand-color-4: $indigo60;
$btn-primary-bg: $brand-color-4;
```

I think we should consider what's the best approach here. One idea is to replace `$brand-color-X` with semantic variables for each use case like `$icon-bg-brand: $indigo60;` and `$icon-bg-brand-disabled: $indigo40;`. Another idea is to keep the `$brand-color-x` variables but rename them to be slightly more descriptive. `$brand-color-primary: $indigo60;` and `$brand-color-hover: $indigo80`. I haven't thought this through 100% yet.

Reply:
I see. Let's see what we come up with as we work through it.